package.json

```diff
@@ -26,7 +26,8 @@
   "files": [
     "*.md",
     "bsconfig.json",
-    "src/*.re"
+    "src/*.re",
+    "src/Interop.js"
```

Comment:
This can probably be more futur proof, just in case.

```suggestion
    "src/*.js",
    "!src/*.bs.js"
```

I just tried using `npm pack --dry` to verify, it looks good :) 

<img width="387" alt="image" src="https://user-images.githubusercontent.com/157534/92885839-6d16fb00-f413-11ea-80c8-35daf9323e3e.png">